### DISCUSSION TITLE
How does "My Languages" work?

### DISCUSSION THREAD

**Discussion by fikotta** [codewars/codewars.com#2711] [Q&A] [answered]
I'm a 4 kyu Python and 6 kyu Javascript, and yet javascript doesn't show up in "my languages" while it does show up on searches designated as "all languages". Why would this be?
![wordle](https://user-images.githubusercontent.com/81991278/171647498-c5c83682-9a55-4d66-9e3f-bc5930e6dcda.png)
![wordle_2](https://user-images.githubusercontent.com/81991278/171647500-20673172-ad77-4f3a-8ba9-5bc3a8dd20a6.png)

**Comment by kazk** [comment, marked as answer]
"My Languages" are the languages you've selected in https://www.codewars.com/trainer/setup
